hle-survey/transformation/src/main/scala/org/broadinstitute/monster/dap/dog/AdditionalStudiesTransformations.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.monster.dap.dog
+
+import org.broadinstitute.monster.dap.RawRecord
+import org.broadinstitute.monster.dogaging.jadeschema.fragment.HlesDogFutureStudies
+
+object AdditionalStudiesTransformations {
+
+  /**
+    * Parse all future-studies health fields out of a raw RedCap record,
+    * injecting them into a partially-modeled dog record.
+    */
+  def mapFutureStudies(rawRecord: RawRecord): HlesDogFutureStudies = {
+    val pcVetConsent = rawRecord.getBoolean("fs_pcvet_consent")
+    HlesDogFutureStudies(
+      fsPrimaryCareVeterinarianExists = rawRecord.getOptionalBoolean("fs_pcvet"),
+      fsPrimaryCareVeterinarianConsentShareVemr = rawRecord.getOptionalBoolean("fs_pcvet_consent"),
+      fsPrimaryCareVeterinarianCanProvideEmail =
+        if (pcVetConsent) rawRecord.getOptionalBoolean("fs_pcvet_email_yn") else None,
```

Comment:
Once `pcVetConsent` is made optional, these will need to be adjusted to type-check. A pattern I've used elsewhere looks like:
```scala
field = pcVetConsent.flatMap {
  if (_) record.getOptional("thing") else None
}
```
You could also use `getOrElse` to default the optional to `false`